src/Domain.LinnApps/GoodsIn/GoodsInService.cs

```diff
@@ -136,5 +197,23 @@ public class GoodsInService : IGoodsInService
 
             return result;
         }
+
+        public ProcessResult ValidatePurchaseOrderQty(
+            int orderNumber, 
+            int qty,
+            int? orderLine)
+        {
+            var valid = this.storesPack.ValidOrderQty(orderNumber, orderLine ?? 1, qty, out _, out _);
```

Comment:
What does out _ do?

Reply:
Nothing really. I think its just a way of denoting that a variable will be unused. I would guess under the hood it stops the runtime allocating memory for that variable